tensorflow/compiler/xla/service/gpu/tests/gpu_codegen_test.cc

```diff
@@ -68,8 +68,14 @@ std::string GpuCodegenTest::MakePlatformSpecificLlvm(absl::string_view input) {
         is_built_with_rocm_ ? "@llvm.amdgcn.s.barrier" : "@llvm.nvvm.barrier0"},
        {"SHUFFLE", is_built_with_rocm_ ? "i32 @llvm.amdgcn.ds.bpermute"
                                        : "float @llvm.nvvm.shfl.sync.down.f32"},
-       {"TIDX", is_built_with_rocm_ ? "llvm.amdgcn.workitem.id.x"
-                                    : "@llvm.nvvm.read.ptx.sreg.tid.x"}});
+       {"TIDX", is_built_with_rocm_ ? "@llvm.amdgcn.workitem.id.x"
+                                    : "@llvm.nvvm.read.ptx.sreg.tid.x"},
+       {"LCAL", is_built_with_rocm_ ? "%[[LOGICAL_T1:.*]] = call { i1, i64 } @llvm.amdgcn.if.i64(i1 %[[LOGICAL_T0]])"
```

Comment:
Can we merge this one and the one below? It seems they always go together?

Reply:
Hmmm....I'm afraid it cannot be. They are two different CHECKs used at different steps for CompilerAndVerifyIr to check. Merge together will change the test order.